finder-app/Makefile

```diff
@@ -0,0 +1,24 @@
+#--------------------------------------
+# Author: Mubeena Udyavar Kazi
+# Course: ECEN 5713 - AESD
+#--------------------------------------
+
+CC := gcc
+
+ifeq ($(CROSS_COMPILE),aarch64-none-linux-gnu-)
```

Comment:
@mubeena12 no need for the more complicated check here, you can just set with $(CROSS_COMPILE)gcc